src/ports.rs

```diff
@@ -0,0 +1,76 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use std::sync::Arc;
+
+use anyhow::Context;
+use enarx_config::{Config, File};
+use static_init::dynamic;
```

Comment:
We already have `lazy_static` https://docs.rs/lazy_static/1.4.0/lazy_static/ in our dependency graph and that's what we use in Enarx. Let's keep using it.